CONTRIBUTING.md

```diff
@@ -109,12 +109,68 @@ Follow GitHub's fork-branch-pull request pattern.
 
 4. For any changes to existing API's, alter the unit tests to cover the changes (and remove tests made irrelevant due to your changes).
 
-5. Review the static code analyses results. 
+5. Review the static code analyses results from the Static Analysis and CodeQL Analysis workflows. For more information on how to use these workflows, view [Using GitHub Actions Workflows](#using-github-actions-workflows).  
     - Push code changes to the appropriate forked repository. 
     - Go to the Actions tab and enable GitHub Actions Workflow. The CodeQL Analyis and Static Analysis will be triggered for all pushed code. 
     - Review these workflows for any warnings or errors. 
     - Once successful, create a pull request. 
 
+### Using GitHub Actions Workflows
+
+Several of our GitHub Actions Workflows are used to ensure pushed code and pull requests do not break cFS, create vulnerabilities, and follow our code conventions. Other workflows are used for documentation purposes. 
+
+Most of the workflows in the NASA/cFS repository will run for all branches when code is pushed and a pull request is created, except for the changelog workflow that runs manually. 
+
+#### View Failed Workflows
+
+1. Navigate to Actions in the selected repository. For newly forked repositories, enable workflows after clicking on Actions. 
+2. In the left sidebar, click the workflow you want to view.
+3. From the list of workflow runs, click the name of the run you want to see.
+4. Under Jobs or in the visualization graph, click the failed job.
+5. Any failed steps are automatically expanded to display the results.
+
+#### View Workflow Results
+1. Navigate to Actions in the selected repository. For newly forked repositories, enable workflows after clicking on Actions. 
+2. In the left sidebar, click the workflow you want to view.
+3. From the list of workflow runs, click the name of the run you want to see.
+4. Scroll to the bottom of the page and download the artifacts. For CodeQL results, navigate to the Security Tab and click Code scanning alerts. CodeQL results will only be avaiable on your forked repository. 
+
+or 
+
+1. In pull requests, click the Checks tab. 
+2. From the list of workflow runs, click the name of the run you want to see.
+4. Scroll to the bottom of the page and download the artifacts. For CodeQL results, expand Code scanning alerts at the bottom of the list of checks and select CodeQL. 
+
+#### Configure Workflows
+
+1. Workflows are under [.github/workflows](https://github.com/nasa/cFS/tree/main/). CodeQL uses a .yml file and a configuration file under .github/codeql(https://github.com/nasa/cFS/tree/main/.github/codeql).
```

Comment:
Fix broken link syntax

Reply:
Move Code QL info to Code QL Analysis entry in the "our workflows" secton

Reply:
Fixed link and moved CodeQL information to the CodeQL Analysis section.